common/util/logging.cc

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017-2020 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace verible {
+int global_vlog_level_ = 0;
```

Comment:
comment about why this exists here? or how it should be used?

Reply:
Done. Mostly nothing user facing, just internally used in the `VLOG()` macro and set in `InitCommandline()`